creator-node/src/blacklistManager.js

```diff
@@ -660,45 +657,6 @@ class BlacklistManager {
     return redis.smembers(redisKey)
   }
 
-  // Delete all existing redis keys
-  static async deleteRedisKeys() {
```

Comment:
you don't think we'll need this code in the future? no idea if we will, just curious